### PR TITLE
Adjustments after changes in Remote Configurations Service

### PR DESCRIPTION
**Pull request by IrinaComindaru**
Issue: https://github.com/jivygroup/mp-reactive-services/issues/315

### CODE REVIEW

File: tests/ConfigurationServiceScalecube/data.test.ts [hunk 1]
```diff
@@ -0,0 +1,45 @@
+import { ConfigurationServiceScalecube } from '../../src/provider/Scalecube';
```
Comment (MikeSquall) [inline tests/ConfigurationServiceScalecube/data.test.ts]: Style: external import should be 1st

File: tests/ConfigurationServiceScalecube/errors.test.ts [hunk 2]
```diff
@@ -0,0 +1,74 @@
+import { ConfigurationServiceScalecube } from '../../src/provider/Scalecube/ConfigurationServiceScalecube';
```
Comment (MikeSquall) [inline tests/ConfigurationServiceScalecube/errors.test.ts]: Style: external import should be 1st

**Review by MikeSquall — APPROVED** [review]
Styling in feature files is still incorrect with excess of indentation

File: tests/ConfigurationServiceScalecube/data.test.ts (hunk 1 above)
Comment (IrinaComindaru) [reply tests/ConfigurationServiceScalecube/data.test.ts]: done

File: tests/ConfigurationServiceScalecube/errors.test.ts (hunk 2 above)
Comment (IrinaComindaru) [reply tests/ConfigurationServiceScalecube/errors.test.ts]: done

**Review by Amir974 — APPROVED** [review]

**Comment by Amir974** [comment]
@idanilt may want to review as well

**Pull request merged**